packages/ember-auto-import/ts/tests/analyzer-test.ts

```diff
@@ -453,10 +453,12 @@ Qmodule('analyzer', function (hooks) {
       await builder.build();
       throw new Error(`expected not to get here, build was supposed to fail`);
     } catch (err) {
-      assert.contains(
-        err.message,
-        'import() is only allowed to contain string literals or template string literals'
-      );
+      if (err instanceof Error) {
```

Comment:
This breaks the test -- if we throw something weird the test will silently pass.

On all of these I think you can say `catch (err: any)` instead.